lib/player.rb

```diff
@@ -21,18 +27,26 @@ def generate(num_balls)
     random_balls.each { |ball| @balls << Ball.new(ball) }
   end
 
+# Method starts with empty arrays because we want to start fresh in between trurns.
```

Comment:
i'm not sure if we want to start fresh every 'turn' but incase you wanted to switch your answer for whatever reason the reset happens because you don't wanna add any extra balls to our ball answer by accident.

Reply:
Thanks, made the changes.